texlive_2022_repos.bzl

```diff
@@ -0,0 +1,6591 @@
+"""
```

Comment:
I'm a bit surprised this isn't the same length as `repositories.bzl` was - I was assuming you renamed it. Is there any way we could preserve the history here, to see what actually changed, by maybe postponing the renaming?

Reply:
Ahh, good point!
Sure, I can move out the logic to a separate file first. Then do a rename?

Reply:
done